web_notify/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+Send instant notification messages to the user in live.
+
+This technical module allows you to send instant notification messages from the server to the user in live.
+Two kinds of notification are supported.
```

Comment:
Not a question regarding the migration but what are these two kinds? Is it meant to be five listed below?

Reply:
Yes, that is an error. Fixed in migration commit.